pages/category/index.vue

```diff
@@ -385,10 +530,11 @@ export default {
   }
 
   &__aside {
-    padding: 0;
     flex: 1;
+    padding: 0;
     @include for-mobile {
```

Comment:
default design should be for mobile-only, hence remove @include for-mobile

Reply:
Resolved